### DISCUSSION TITLE
When will I finally get Github Copilot access ?

### DISCUSSION THREAD

**Discussion by f0-x** [community/community#16584] [Copilot Conversations]
I have been an avid follower of the Github Copilot and have been on the waitlist for many months now. All of my colleagues at work have it and even the juniors, interns(basically everyone but me) do. Is there any process to speed up my chances to get the chance to experience the power of Github Copilot, via purchase maybe ? Thanks.

**Comment by asrvd** [comment]
#9342 may help you.
